app/assets/javascripts/modules/track-radio-group.js

```diff
@@ -1,95 +1,104 @@
 window.GOVUK = window.GOVUK || {}
-window.GOVUK.Modules = window.GOVUK.Modules || {};
+window.GOVUK.Modules = window.GOVUK.Modules || {}
 
-(function (global, GOVUK) {
-  'use strict'
+window.GOVUK.TrackRadioGroupVerify = function (data) {
+  window.GOVUK.triggerEvent(document, 'have-verify-data', { detail: data })
+};
 
-  var $ = global.jQuery
+(function (Modules) {
+  function TrackRadioGroup (element) {
+    this.element = element
+  }
 
-  GOVUK.Modules.TrackRadioGroup = function () {
-    this.start = function (element) {
-      track(element)
+  TrackRadioGroup.prototype.init = function () {
+    document.addEventListener('have-verify-data', function (event) {
+      this.reportVerifyUser(this.element, event.detail)
+    }.bind(this))
 
-      if (crossDomainTrackingEnabled(element)) {
-        addCrossDomainTracking(element)
-      }
+    this.track(this.element)
 
-      checkVerifyUser(element)
+    if (this.crossDomainTrackingEnabled(this.element)) {
+      this.addCrossDomainTracking(this.element)
     }
 
-    function track (element, withHint) {
-      element.on('submit', function (event) {
-        var options = { transport: 'beacon' }
+    this.checkVerifyUser(this.element)
+  }
 
-        var $submittedForm = $(event.target)
+  TrackRadioGroup.prototype.track = function (element, withHint) {
+    this.element.addEventListener('submit', function (event) {
+      var options = { transport: 'beacon' }
+      var submittedForm = event.target
+      var checkedOption = submittedForm.querySelector('input:checked')
+      var eventValue = this.eventTrackingValue(checkedOption, withHint)
 
-        var $checkedOption = $submittedForm.find('input:checked')
+      GOVUK.analytics.trackEvent('Radio button chosen', eventValue, options)
 
-        var eventValue = eventTrackingValue($checkedOption, withHint)
+      if (this.crossDomainTrackingEnabled(submittedForm)) {
+        this.trackCrossDomainEvent(submittedForm, eventValue, options)
+      }
+    }.bind(this))
+  }
 
-        GOVUK.analytics.trackEvent('Radio button chosen', eventValue, options)
+  TrackRadioGroup.prototype.checkVerifyUser = function (element) {
```

Comment:
Maybe a little comment here would be helpful to explain what this function is doing. Maybe

```
// the analytics data should include whether the user has previously signed in to Verify
// this simulates the behaviour of JSONP, returning a true or false value from Verify
```